onnx/defs/schema.h

```diff
@@ -463,13 +463,40 @@ class OpSchema {
   };
 };
 
+// Map type to store operator schemas. The format is,
+// <OpName, <Domain, <OperatorSetVersion, OpSchema>>>.
```

Comment:
I didn't catch this when I glanced through this patch originally, but having the map layer be operator name, and then domain, doesn't seem right. Operators with the same name in different domains are completely not related to each other, so I don't see why you should first case on op name, and then domain.

Reply:
hmmm, <op_name, domain> or <domain, op_name> both are fine, right?